test/fixtures/globals.ts

```diff
@@ -1,6 +1,18 @@
 import {JSDOM} from 'jsdom';
 
 const {window} = new JSDOM('...');
+
+// Extending global to recognize the new properties we're adding on.
+declare global {
```

Comment:
I did the declaration merging in here because it's really isolated to this file. If you want me to pull it out into it's own `*.d.ts` file, let me know.